.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
I find 60 days a relatively short amount of time. I could imagine that issues won't get tackled for a quarter and maybe only get more activity / re-triaged after a quarter of work has been done and we go on to the next things to do? So maybe 90-100 days may be more safe to really make sure the issue hasn't been considered for more than a quarter?

Reply:
idea: could leave the stale at 60 and change `daysUntilClose` to 30?

Reply:
Then again its marked as stale and doesn't get a response within a certain window it probably never will. Maybe 14 days to allow a bit more time would be the middle ground?

Reply:
Given that PTO can (and should… to get a reasonable amount of breathe time) be > 14 days, I'd also vote for 30 days as someone (the author) being disconnected for more than 30 days is probably rare.

Obviously, the author can reply to an closed issue and we can handle that too… not sure what feels best, I think I'd go for @eviljeff's idea

Reply:
On the basis that an issue can still be re-opened I don't see too much of an issue with shorter times overall. This might be a case of trying it and adapting the rules if they feel like they're not working. Also could be interesting to see what bugzilla's settings are for things like this if we can find them.

Reply:
Looks like I'm in the minority here... I was thinking of having a much higher delay like 180 days! While issues can still be re-opened, it's slightly harder to find issues that have been deleted (more noise to go through).

Reply:
> Looks like I'm in the minority here... I was thinking of having a much higher delay like 180 days! While issues can still be re-opened, it's slightly harder to find issues that have been deleted (more noise to go through).

I've no problem with the initial delay being longer. 6 months seems ok for what we want this for.

Reply:
I've updated  `daysUntilStale: 180` and `daysUntilClose: 14`

Does that seem ok to start?